.identity/00_variables.tf

```diff
@@ -0,0 +1,53 @@
+variable "tags" {
+  type = map(any)
+}
+
+variable "prefix" {
+  type = string
+}
+
+variable "env" {
+  type = string
+}
+
+variable "env_short" {
+  type = string
+}
+
+variable "domain" {
+  type = string
+}
+
+variable "ci_github_federations" {
+  type = list(object({
+    repository        = string
+    credentials_scope = optional(string, "environment")
+    subject           = string
+  }))
+  description = "GitHub Organization, repository name and scope permissions"
+}
+
+variable "opex_environment_ci_roles" {
+  type = object({
+    subscription    = list(string)
+    resource_groups = map(list(string))
+  })
+  description = "GitHub Continous Integration roles"
+}
+
+variable "cd_github_federations" {
```

Comment:
Not used anymore, can be deleted

Reply:
done in [14ee036](https://github.com/pagopa/io-web-profile-backend/pull/32/commits/14ee0365658a58dfc25fe0c9fbb78d25f4fb2503)